src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,17 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform 300 ms color 300ms;
```

Comment:
You'd better remove space between 300 and ms: `300 ms`.
It seems like it doesn't work.